packages/card/src/CardActions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import CardRow from './CardRow';
+import cardTheme from './cardTheme';
+
+/**
+ * Card actions component
+ */
+export default createStyledComponent(
+  CardRow,
+  (props, baseTheme) => {
+    const theme = cardTheme(baseTheme);
+
+    return {
+      paddingLeft: theme.CardActions_padding,
+      paddingRight: theme.CardActions_padding,
+
+      '&:last-child': {
+        marginBottom: 0,
+        marginTop: theme.CardActions_marginTop
+      }
+    };
+  },
+  {
+    displayName: 'CardActions'
```

Comment:
All of the subcomponents are only intended to be used within the root Card component. All of the reset styles, with the exception of `outline`, inherit... so we should be fine as-is.